public/index.html

```diff
@@ -5,25 +5,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
-    <title>Police Abuses: Hundreds of Videos of Police Brutality from George Floyd Protests</title>
-    <meta name="description" content="A compilation of all of the police brutality incidents during the George Floyd protests. Share videos, find videos, and help hold police officers accountable."/>
+    <title><%=VUE_APP_TITLE %></title>
     <!-- Twitter Card data -->
     <meta name="twitter:card" content="summary">
-    <meta name="twitter:site" content="http://policeabuses.com">
-    <meta name="twitter:title" content="PoliceAbuses: 200+ Videos of Police Brutality From George Floyd Protests">
-    <meta name="twitter:description" content="A compilation of all of the police brutality incidents during the George Floyd protests. Share videos, find videos, and help hold police officers accountable.">
```

Comment:
This is the only description that will work without server-side rendering so might be better to keep it around.

Reply:
I don't understand your feedback. These attributes are being set based on the incidents once vue kicks it.